ipykernel/kernelbase.py

```diff
@@ -855,10 +856,26 @@ async def debug_request(self, stream, ident, parent):
         if inspect.isawaitable(reply_content):
             reply_content = await reply_content
         reply_content = json_clean(reply_content)
+        reply_content['hello'] = 'hello'
```

Comment:
Was this left in by accident?

Reply:
Thx. 0c8fcba479a60d299cfea3489300beb0217b4cd9 removes this.